src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
```

Comment:
The template literal syntax is incorrect here. In JSX, you should use curly braces for expressions. Change `${person.name}` to `{person.name}`.